core/services/relay/relay.go

```diff
@@ -0,0 +1,27 @@
+package relay
+
+import (
+	"github.com/smartcontractkit/chainlink/core/service"
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+type Relays struct {
+	// TODO: support Ethereum as a relay
+	// Ethereum Relay
+	Solana Relay
```

Comment:
see comment above, I think it's a cleaner separation if the ocr relay jobs doesn't know anything about available chains/what chain its relaying to. We could have single object implement Relay and based on NewOCR2Service arguments return the appropriate relay from the relay specific package (like the solana one you have here). Wdyt?

Reply:
Please check out the latest implementation. There is now a master relayer, a delegate abstracting all available implementations (Ethereum & mocked Solana, for now).